bindings/CMakeLists.txt

```diff
@@ -25,9 +25,13 @@ if(pyroot)
     add_subdirectory(tpython)
     add_subdirectory(jupyroot)
     add_subdirectory(jsmva)
-    if(dataframe AND (PYTHON_VERSION_STRING_Development_Main VERSION_GREATER_EQUAL 3.7))
-      # TODO: Add a status message to signal both if DistRDF is being built or if it isn't
+    if(dataframe AND (PYTHON_VERSION_STRING_Development_Main VERSION_GREATER_EQUAL 3.8))
+      message(STATUS "Distributed RDataFrame enabled")
       add_subdirectory(experimental/distrdf)
+    else()
+      message(STATUS "Missing features to enable distributed RDataFrame:")
```

Comment:
Instead of "missing features" I'd say "requirements"? Then below you specify what's the required value and what's the actual one.